app/controllers/two_factor_authentication/recovery_code_verification_controller.rb

```diff
@@ -13,15 +13,15 @@ def create
       @recovery_code_form = RecoveryCodeForm.new(current_user, personal_key)
       result = @recovery_code_form.submit
 
-      analytics.track_event(Analytics::MULTI_FACTOR_AUTH, result.merge(method: 'recovery code'))
+      analytics.track_event(Analytics::MULTI_FACTOR_AUTH, result.to_h)
```

Comment:
this isn't necessary for now, but in the future it might be helpful to make these named args